ui/lib/kv/addon/routes/list-directory.js

```diff
@@ -61,8 +62,18 @@ export default class KvSecretsListRoute extends Route {
 
   setupController(controller, resolvedModel) {
     super.setupController(controller, resolvedModel);
+    // listPermissions is used to show the overview card and/or an error message about LIST permissions.
+    resolvedModel.listPermissions = {
+      noDirectory: false,
+      noList: false,
```

Comment:
is `noList` only for the top level of a secret? and `noDirectory` only for sub secrets and could be for whether a secret exists or no permissions?